server/src/main/java/io/spine/server/aggregate/DefaultAggregatePartRepository.java

```diff
@@ -18,13 +18,35 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-package io.spine.system.server.given.command;
+package io.spine.server.aggregate;
 
-import io.spine.server.aggregate.AggregateRepository;
-import io.spine.system.server.CompanyId;
+import io.spine.server.aggregate.model.AggregatePartClass;
+
+import static io.spine.server.aggregate.model.AggregatePartClass.asAggregatePartClass;
 
 /**
- * A repository of {@link CompanyAggregate}.
+ * Default implementation of {@code AggregatePartRepository}.
  */
-public final class CompanyRepository extends AggregateRepository<CompanyId, CompanyAggregate> {
+public final class DefaultAggregatePartRepository<I,
```

Comment:
Please consider making all the default repositories `@Internal`. I don't recognize this as a part of the public API.